packages/cursorless-engine/src/core/commandRunner/CommandRunnerImpl.ts

```diff
@@ -35,36 +41,15 @@ export class CommandRunnerImpl implements CommandRunner {
    *    action, and returns the desired return value indicated by the action, if
    *    it has one.
    */
-  async run({
-    action: { name: actionName, args: actionArgs },
-    targets: partialTargetDescriptors,
-  }: CommandComplete): Promise<unknown> {
-    checkForOldInference(partialTargetDescriptors);
-    const targetDescriptors = inferFullTargets(partialTargetDescriptors);
-
-    if (this.debug.active) {
-      this.debug.log("Full targets:");
-      this.debug.log(JSON.stringify(targetDescriptors, null, 3));
-    }
-
-    const action = this.actions[actionName];
-
-    const prePositionStages =
-      action.getPrePositionStages?.(...actionArgs) ?? [];
-    const finalStages = action.getFinalStages?.(...actionArgs) ?? [];
-
-    const targets = targetDescriptors.map((targetDescriptor) =>
-      this.pipelineRunner.run(targetDescriptor, prePositionStages, finalStages),
-    );
-
+  async run({ action }: CommandComplete): Promise<unknown> {
```

Comment:
Huh if this is all you need to run a command I wonder if we should rename it 🤔. Or inline it. Let's discuss

Reply:
I kinda like the separation between the running the action and the rest of the command.

Reply:
I guess I'm not opposed to that, tho it breaks our nomenclature a bit, because "action" usually refers to the action without its targets, whereas "command" is the full command including targets.  Eg in the cheatsheet we have a table of "actions".  Let's discuss at meet-up.  I think just a better name here is fine, because inlining them is slightly more awkward because then all the different versions need to extend a base